package.json

```diff
@@ -29,35 +29,42 @@
     "@babel/preset-env": "^7.12.1",
     "@babel/preset-react": "^7.12.1",
     "@babel/preset-typescript": "^7.12.1",
+    "@types/axios": "^0.14.0",
     "@types/cors": "^2.8.8",
     "@types/express": "^4.17.8",
     "@types/jest": "^26.0.14",
+    "@types/memcached": "^2.2.6",
```

Comment:
**Comment**: Please audit and remove any unused packages from `package.json`.

Reply:
We will be using memcached as a temporary database correct?

Reply:
Possibly; we will have to determine if it's appropriate for our use case. Even then, we should not be installing new packages before they are strictly ready to be used. Recall the old programming adage: https://en.wikipedia.org/wiki/You_aren't_gonna_need_it

Reply:
Uninstalled @types/memcached and memcached.